packages/onboarding/src/intent-screen/README.md

```diff
@@ -0,0 +1,40 @@
+# IntentScreen
+
+Currently used in WordPress.com signup flow. This step gives the user a few different starting points to choose from and an additional alternative at the bottom.
+
+## Props
+
+| Name            | Type            | Required | Description                                                                                  |
+| --------------- | --------------- | -------- | -------------------------------------------------------------------------------------------- |
+| `intents`       | `SelectItem`    | yes      | Main items listed with text and buttons. See type description below.                         |
+| `intentsAlt`    | `SelectAltItem` | yes      | Alternative option styled without a button and placed at the end. See type description below.|
+| `onSelect`      | `function`      | yes      | ( **value** ) Callback to run when an item is selected. This should have no return.          |
+| `preventWidows` | `function`      | yes      | ( **text, wordsToKeep** - _optional_ )  Wrapper for the _intents_ title and description.     |
+
+## SelectItems
+
+Items used in the IntentScreen. They consist of a title, description, and icon.
+
+### Props
+
+- **key** _string_ - unique identifier for the item.
+- **title** _string_ - text to use as the header for the item.
+- **description** _string_ - text explaining the item, not translated or wrapped.
+- **icon**: _ReactElement_ - uses WordPress _Icon_ component and sets it to 24px
+- **value**: _string_ - passed to onClick/onSelect handler to identify the item.
+- **actionText**: _sring_ - text for the button.
+- **hidden**: _boolean_ - if true, item is hidden.
+
+## SelectAltItem
+
+This is the final item in IntentScreen and does not have a button or icon.
+
+### Props
```

Comment:
We made a wrong assumption when migrating `SelectItems` and `SelectItemsAlt` that they were part of `IntentScreen`. Turns out they're actually not. I didn't mention modifying our PRs because it's an implementation detail (you import them like `import { SelectItems } from 'onboarding'`, so our mistake is hidden). But this makes it official. 

`SelectItems` are just selectable items. Not necessarily part of intent screen.

We'll come around to actually fixing them because the CSS selectors are still affected.

Reply:
Updated this in #61770 and removed this from the README